__test__/api/replies/deleteSinglereply.test.js

```diff
@@ -0,0 +1,149 @@
+const app = require("../../../server");
+const supertest = require("supertest");
+const ReplyModel = require("../../../models/replies");
+const CommentModel = require("../../../models/comments");
+const commentHandler = require("../../../utils/commentHandler");
+const replyHandler = require("../../../utils/replyHandler");
+const request = supertest(app);
+
+describe("DELETE /comments/:commentId/replies/:replyId", () => {
+  let comment;
+  let reply;
+
+  beforeEach(async () => {
+    //dummy comment document
+    const dummyComment = new CommentModel({
+      content: "A comment from user 1",
+      ownerId: "user1@email.com",
+      origin: "123123",
+      refId: 2,
+      applicationId: global.application._id,
+    });
+    //dummy replies document
+    const dummyReply = new ReplyModel({
+      content: "A reply from user 2",
+      ownerId: "user2@email.com",
+      commentId: dummyComment.id,
+    });
+
+    //save dummy comment,reply
+    const savedC = await dummyComment.save();
+    const savedR = await dummyReply.save();
+
+    // Add replies to the dummy comment
+    await CommentModel.findByIdAndUpdate(savedC.id, { replies: savedR.id });
```

Comment:
this does not correctly add `replyId` to `comment.replies`
review your work or use 
```
const savedR = await dummyReply.save();
dummyComment.replies.push(savedR.id);
const savedC = await dummyComment.save();
```